rfcs/text/0005-host-metric-fields.md

```diff
@@ -125,6 +125,13 @@ Stage 4: Document any new concerns and their resolution. The goal here is to eli
 <!--
 Stage 4: Identify at least one real-world, production-ready implementation that uses these updated field definitions. An example of this might be a GA feature in an Elastic application in Kibana.
 -->
+Our goal is to switch related fields to these new host metrics in Kibana Observability
+metrics UI. For example: right now under hosts inventory, CPU usage will only display
+metrics that are collected by Metricbeat `system` module. With using the new host
+metric fields, CPU metric from `system` module will be `host.cpu.usage`, as well
+as CPU metrics from all AWS EC2 instances or Azure compute VMs. With Kibana Metrics
+UI switching to these new proposed host metric fields, all hosts will be discoverd
```

Comment:
Any issue(s) we should link as a reference for this upcoming switch?

Reply:
Hi @ebeahan, thanks for the review. I don't think we have any issues right now to track this work. I will create them and update here.

Reply:
Hi @ebeahan, sorry for the delay. I just added the link for switching to new field names in Kibana Metrics UI.